x-pack/plugins/observability_solution/exploratory_view/public/plugin.ts

```diff
@@ -138,7 +143,10 @@ export class Plugin
     return {
       createExploratoryViewUrl,
       getAppDataView: getAppDataView(pluginsStart.dataViews),
-      ExploratoryViewEmbeddable: getExploratoryViewEmbeddable({ ...coreStart, ...pluginsStart }),
+      ExploratoryViewEmbeddable: getExploratoryViewEmbeddable(
+        { ...coreStart, ...pluginsStart },
```

Comment:
The `this.analyticsService` should be part of coreStart?

Reply:
well it's coming from coreSetup, not coreStart.